bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/security.xml

```diff
@@ -197,6 +197,7 @@
 	<channel-group-type id="presence">
 		<label>Presence Camera</label>
 		<channels>
+			<channel id="siren" typeId="siren-status-rw"/>
```

Comment:
Can you use the new thing upgrade mechanism to automatically add this new channel in the existing things ?

Reply:
This will be another beast because I do not use xml defined things in this binding.